maquetado/setup/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+$tech-logo-size: 100px;
+$tech-container-margin: 50px;
+$tech-container-height: 300px;
+$tech-container-width: 210px;
+
+h2 {
+  font-size: $text-medium;
+}
+
+.tech {
+  align-items: center;
+  background-color: $green;
+  border: 7px;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: column wrap;
+  height: $tech-container-height;
+  left: 60px;
+  margin: $tech-container-margin;
+  position: relative;
+  width: $tech-container-width;
+
+  &::before {
+    border: 1px solid;
+    border-color: $green;
+    border-radius: 5px;
+    content: '';
+    height: $tech-container-height;
+    margin: $tech-container-margin;
+    position: absolute;
+    right: -45px;
```

Comment:
I think this numbers are too big 🤔  They should be something like `-10 px`, try with `top` and `left`